PEC_ReceivedNotification/index.html

```diff
@@ -161,5 +161,23 @@
+                  background-color: #0073e6;
+                  color: #fff !important;
+                  padding: 8px 20px;
+                  border-radius: 4px;
+                  text-decoration: none;" title="Visualizza i documenti" class="button" href="${quickloginAccessLink}"><strong>Visualizza i documenti su SEND</strong></a></p><!-- SEND --><p style="font-size: 14px; line-height: 20px">Ricevi questo avviso di avvenuta ricezione in qualit&agrave; di persona ${recipientType} con Codice Fiscale ${recipient.taxId}. SEND &egrave; uno strumento che rende l&apos;invio di notifiche da parte della Pubblica Amministrazione pi&ugrave; sicuro, efficiente ed economico. <a href="${PNFaqSendURL}" title="Scopri di pi&ugrave;">Scopri di pi&ugrave;</a></p></div></td></tr><tr><td align="left" class="text" style="font-size:0px;padding:10px 25px;padding-bottom:0px;word-break:break-word;"><div style="font-family:Titillium Web, system-ui, sans-serif;font-size:14px;line-height:20px;text-align:left;color:#17324D;"><p style="margin-top: 0px">A presto,<br aria-hidden="true">SEND - Servizio Notifiche Digitali</p></div></td></tr></tbody></table></div><!--[if mso | IE]></td></tr></table><![endif]--></td></tr></tbody></table></div><!--[if mso | IE]></td></tr></table></td></tr><![endif]--><!-- Divider --><!--[if mso | IE]><tr><td class="section-outlook" width="600px" ><table align="center" border="0" cellpadding="0" cellspacing="0" class="section-outlook" role="presentation" style="width:600px;" width="600" ><tr><td style="line-height:0px;font-size:0px;mso-line-height-rule:exactly;"><![endif]--><div class="section" style="margin:0px auto;max-width:600px;"><table align="center" border="0" cellpadding="0" cellspacing="0" role="presentation" style="width:100%;"><tbody><tr><td style="direction:ltr;font-size:0px;padding:20px 0;padding-bottom:0px;padding-top:0px;text-align:center;"><!--[if mso | IE]><table role="presentation" border="0" cellpadding="0" cellspacing="0"><tr><td class="" style="vertical-align:top;width:600px;" ><![endif]--><div class="mj-column-per-100 mj-outlook-group-fix" style="font-size:0px;text-align:left;direction:ltr;display:inline-block;vertical-align:top;width:100%;"><table border="0" cellpadding="0" cellspacing="0" role="presentation" style="vertical-align:top;" width="100%"><tbody><tr><td align="center" style="font-size:0px;padding:10px 25px;padding-top:16px;padding-bottom:16px;word-break:break-word;"><p style="border-top:solid 1px lightgrey;font-size:1px;margin:0px auto;width:100%;"></p><!--[if mso | IE]><table align="center" border="0" cellpadding="0" cellspacing="0" style="border-top:solid 1px lightgrey;font-size:1px;margin:0px auto;width:550px;" role="presentation" width="550px" ><tr><td style="height:0;line-height:0;"> &nbsp;
```

Comment:
@SarahDonvito in this PR I see a different variable [https://github.com/pagopa/pn-delivery-push/pull/894/files#diff-84693e95fd5b276c5412ab2836d5ff67f6dcecf7f8c236472322ec94b948b41e]( https://github.com/pagopa/pn-delivery-push/pull/894/files#diff-84693e95fd5b276c5412ab2836d5ff67f6dcecf7f8c236472322ec94b948b41e)
which one is correct?